prepare/tasks/qa/tasks.py

```diff
@@ -25,7 +25,7 @@
 add_to_catalog(
     FormTask(
```

Comment:
We should change all QA tasks to support "answers" and not "answer.  Right now it will not be consistent.
add_to_catalog(
    FormTask(
        inputs=["context", "context_type", "question"],
        outputs=["answer"],
        metrics=["metrics.squad"],
    ),
    "tasks.qa.with_context.extractive",
    overwrite=True,
)

add_to_catalog(
    FormTask(
        inputs=["context", "context_type", "question"],
        outputs=["answer"],
        metrics=["metrics.rouge"],
        augmentable_inputs=["context", "question"],
    ),
    "tasks.qa.with_context.abstractive",
    overwrite=True,
)

Reply:
Done!

Reply:
Note that if we change "tasks.qa.with_context.abstractive" to "answers", then we need to fix all the cards that us it.

Reply:
yes, and change some templates too.